### PR TITLE
Update the Istio security tasks for V0.8.

### PR DESCRIPTION
**Pull request by myidpt**
* Remove the Istio Citadel service from the health check task because it is by default defined.
* Update the clean up steps.

### CODE REVIEW

**Comment by myidpt** [comment]
@wattli Sure.

File: content/docs/tasks/security/health-check.md [hunk 1]
```diff
@@ -26,9 +26,15 @@ this feature is not needed if the production setup is not using the
 ## Before you begin
 
 * Set up Istio by following the instructions in the
-  [quick start](/docs/setup/kubernetes/quick-start/).
-  Note that authentication should be enabled at step 5 in the
-  [installation steps](/docs/setup/kubernetes/quick-start/#installation-steps).
+  [quick start](/docs/setup/kubernetes/quick-start/) with global mTLS enabled:
```
Comment (geeknoid) [inline content/docs/tasks/security/health-check.md]: mTLS -> mutual TLS

File: content/docs/tasks/security/health-check.md [hunk 2]
```diff
@@ -26,9 +26,15 @@ this feature is not needed if the production setup is not using the
...
+
+    ```command
+    $ kubectl apply -f install/kubernetes/istio-demo-auth.yaml
+    ```
+    _**OR**_
+    Using [Helm](/docs/setup/kubernetes/helm-install/) with `global.mtls.enabled` to `true`.
+
+> Starting with Istio  0.7, you can use [authentication policy](/docs/concepts/security/authn-policy/) to config mTLS for all/selected services in a namespace (repeated for all namespaces to get global setting). See [authentication policy task](/docs/tasks/security/authn-policy/)
```
Comment (geeknoid) [inline content/docs/tasks/security/health-check.md]: config -> configure
mTLS -> mutual TLS

File: content/docs/tasks/security/mutual-tls.md [hunk 3]
```diff
@@ -48,7 +48,7 @@ Citadel is up if the "AVAILABLE" column is 1.
 
 ### Verifying service configuration
 
-* Check installation mode. If mutual TLS is enabled by default (e.g `istio-auth.yaml` was used when installing Istio), you can expect to see uncommented `authPolicy: MUTUAL_TLS` in configmap.
+* Check installation mode. If mutual TLS is enabled by default (e.g `istio-demo-auth.yaml` was used when installing Istio), you can expect to see uncommented `authPolicy: MUTUAL_TLS` in configmap.
```
Comment (geeknoid) [inline content/docs/tasks/security/mutual-tls.md]: configmap -> the configmap

File: content/docs/tasks/security/plugin-ca-cert.md [hunk 4]
```diff
@@ -13,9 +13,15 @@ operator-specified root certificate. This task demonstrates an example to plug c
 ## Before you begin
 
 * Set up Istio by following the instructions in the
-  [quick start](/docs/setup/kubernetes/quick-start/).
-  Note that authentication should be enabled at step 5 in the
-  [installation steps](/docs/setup/kubernetes/quick-start/#installation-steps).
+  [quick start](/docs/setup/kubernetes/quick-start/) with global mTLS enabled:
```
Comment (geeknoid) [inline content/docs/tasks/security/plugin-ca-cert.md]: mTLS -> mutual TLS

File: content/docs/tasks/security/plugin-ca-cert.md [hunk 5]
```diff
@@ -13,9 +13,15 @@ operator-specified root certificate. This task demonstrates an example to plug c
...
+
+    ```command
+    $ kubectl apply -f install/kubernetes/istio-demo-auth.yaml
+    ```
+    _**OR**_
+    Using [Helm](/docs/setup/kubernetes/helm-install/) with `global.mtls.enabled` to `true`.
+
+> Starting with Istio  0.7, you can use [authentication policy](/docs/concepts/security/authn-policy/) to config mTLS for all/selected services in a namespace (repeated for all namespaces to get global setting). See [authentication policy task](/docs/tasks/security/authn-policy/)
```
Comment (geeknoid) [inline content/docs/tasks/security/plugin-ca-cert.md]: config -> configure
mTLS -> mutual TLS

File: content/docs/tasks/security/health-check.md (hunk 1 above)
Comment (myidpt) [reply content/docs/tasks/security/health-check.md]: Done.

File: content/docs/tasks/security/health-check.md (hunk 2 above)
Comment (myidpt) [reply content/docs/tasks/security/health-check.md]: Done.

File: content/docs/tasks/security/mutual-tls.md (hunk 3 above)
Comment (myidpt) [reply content/docs/tasks/security/mutual-tls.md]: Done.

File: content/docs/tasks/security/plugin-ca-cert.md (hunk 4 above)
Comment (myidpt) [reply content/docs/tasks/security/plugin-ca-cert.md]: Done.

File: content/docs/tasks/security/plugin-ca-cert.md (hunk 5 above)
Comment (myidpt) [reply content/docs/tasks/security/plugin-ca-cert.md]: Done.

**Comment by geeknoid** [comment]
Please rebase and resubmit in order to resolve the conflicts.

**Comment by geeknoid** [comment]
Deploy preview for *preliminary-istio* failed.

Built with commit edb18a6b0a4659ec6520a918945d470908abab50

https://app.netlify.com/sites/preliminary-istio/deploys/5b1823321f12b7670ccffff0

**Pull request merged**